---
title: MSSQLSERVER_8680 | Microsoft Docs
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.service: ''
ms.component: errors-events
ms.reviewer: ''
ms.suite: sql
ms.technology:
- database-engine
ms.tgt_pltfrm: ''
ms.topic: language-reference
helpviewer_keywords:
- 8680 (Database Engine error)
ms.assetid: 79f7270e-e4df-4f4e-bc78-2dcad6a9f140
caps.latest.revision: 13
author: edmacauley
ms.author: edmaca
manager: craigg
ms.workload: Inactive
ms.openlocfilehash: e27c6018144510fa542ef61dcb7e9a92cc289f3a
ms.sourcegitcommit: 7a6df3fd5bea9282ecdeffa94d13ea1da6def80a
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 04/16/2018
---
# <a name="mssqlserver8680"></a>MSSQLSERVER_8680
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|8680|  
|イベント ソース|MSSQLSERVER|  
|コンポーネント|SQLEngine|  
|シンボル名|REMOTE_ERR|  
|メッセージ テキスト|内部クエリ プロセッサ エラー: リモート クエリ フェーズを処理中に、クエリ プロセッサで予期しないエラーが発生しました。|  
  
## <a name="explanation"></a>説明  
クエリ処理中に内部エラーが発生しました。  
  
## <a name="user-action"></a>ユーザーの操作  
マイクロソフト カスタマー サポート サービス (CSS) に連絡し、エラーの説明とトレースまたはダンプ (取得可能な場合) を報告してください。  
  
